examples/redis/ansible/tasks/redis_variables.yml

```
@@ -1,5 +1,7 @@
 - name: Set variables into redis database
-  ansible.builtin.shell: |
+  ansible.builtin.command: |
     redis-cli set key value1
+  register: output
+  changed_when: output.rc != 0
```

While for example it does not really matter, I would expect this to be

```suggestion
  changed_when: output.rc != 0
```
The task is changed if redis successfully made the change, and I expect it in that case to return 0

---

maybe it would be simpler just use `changed_when: false` to indicate this is irrelevant?

---

Use `changed_when: false` instead.